modules/imap/functions.php

```diff
@@ -318,8 +322,8 @@ function format_imap_message_list($msg_list, $output_module, $parent_list=false,
                     array('checkbox_callback', $id),
                     array('safe_output_callback', 'source', $source, $icon),
                     array('safe_output_callback', 'from'.$nofrom, $from, null, str_replace(array($from, '<', '>'), '', $msg['from'])),
-                    array('subject_callback', $subject, $url, $flags, null, $preview_msg),
```

Comment:
Why do you remove the preview message parameter pass here? Doesn't seem related with the functionality you are adding.